### PR TITLE
Check Role networkAttachments

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml [hunk 1]
```diff
@@ -135,5 +135,10 @@ spec:
           edpm_hosts_entries_vip_hosts_entries: []
           hosts_entries: []
           hosts_entry: []
+      networkAttachments:
+        - ctlplane
+        - internalapi
+        - storage
+        - tenant
```
Comment (fao89) [inline config/samples/dataplane_v1beta1_openstackdataplane.yaml:142]: I noticed NETWORKATTACHMENTS were empty
```
oc get osaee -n openstack   
NAME                                            NETWORKATTACHMENTS   STATUS   MESSAGE
dataplane-deployment-configure-networkndjhw                          True     AnsibleExecutionJob complete
dataplane-deployment-configure-openstacktspcd                        True     AnsibleExecutionJob complete
dataplane-deployment-configure-osb58ch                               True     AnsibleExecutionJob complete
dataplane-deployment-install-openstackwl7qv                          True     AnsibleExecutionJob complete
dataplane-deployment-install-oswdn5w                                 True     AnsibleExecutionJob complete
dataplane-deployment-run-openstackpklrh                              True     AnsibleExecutionJob complete
dataplane-deployment-run-oswh795                                     True     AnsibleExecutionJob complete
dataplane-deployment-validate-network2z6vr                           True     AnsibleExecutionJob complete
```
@stuggi is this the right way to use `networkAttachments`?

Comment (stuggi) [reply config/samples/dataplane_v1beta1_openstackdataplane.yaml:142]: @fao89 right now the implementation in the dataplaneoperator for networkattachments [1] is only on a per node base. we probably want to enhance this and also add it as a per role, or on openstackdataplane top level parameter and pass it down.

[1] https://github.com/openstack-k8s-operators/dataplane-operator/pull/53

**Comment by fao89** [comment]
I've tested and now I can see it:
```
oc get osaee -n openstack
NAME                                            NETWORKATTACHMENTS                              STATUS   MESSAGE
dataplane-deployment-configure-networkx2879     ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-configure-openstackh8ndh   ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-configure-ostdrlv          ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-install-openstacktwx6d     ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-install-osmzb7f            ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-run-openstack2wffb         ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-run-osfshdm                ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-validate-network9xkfh      ["ctlplane","internalapi","storage","tenant"] 
```

**Pull request closed without merging**